src/content.ts

```diff
@@ -196,6 +196,12 @@ export class Content {
       title,
       prompt,
     });
+
+    if (res.status !== 200) {
+      const message = res.status === 400 ? await res.text() : res.statusText;
```

Comment:
Is this how we serialize errors for other routes? We generally return JSON.

Reply:
Yes, that's how we do it in most places - actually until recently we were just including HTTP status message. 
Just including the entire response in the exception message is less brittle than trying to parse JSON error object, because our errors are pretty heterogeneous and changeable. For example, adopting schema validation for a certain API can change the format of the error object we return.

Reply:
I agree it's not brittle. But I would also consider errors (especially 400) part of the API contract. But if we are doing this other places, that's a problem we can solve some other day.

Reply:
Yeah next time we're touching this, we should probably centralize this logic in a helper method so we won't need to patch dozens of places, too.